src/common/components/repositories-list/index.js

```diff
@@ -5,9 +5,9 @@ import { conf } from '../../helpers/config';
 import { createSnap } from '../../actions/create-snap';
```

Comment:
I think it shows repositories that have had snap building enabled, not snaps as such.